helsingborg-stad/app-mitt-helsingborg
# Orphan/onboarding

## Explain changes
Complete rewrite of on boarding.
No other component should have been affected or changed.

## How to test the changes?
Check out the branch and then check out the smooth animations, the crisp colors and the cool layouts.

## Was this feature tested in the following environments?
